### PR TITLE
Uses parallel construction in the COI section

### PR DESCRIPTION
**Pull request by eliocamp**
This small PR changes the wording in the COI section so that every bulletpoint follows the same parallel construction.

### CODE REVIEW

File: softwarereview_policies.Rmd [hunk 1]
```diff
@@ -35,10 +35,10 @@ Last but not least, you'll find the [code of conduct of rOpenSci Software Peer R
 
 Following criteria are meant to be a guide for what constitutes a conflict of interest for an editor or reviewer. The potential editor or reviewer has a conflict of interest if:
 
-* The authors with a major role are from the potential reviewer/editor's institution or institutional component (e.g., department)
-* Within in the past three years, the potential reviewer/editor has been a collaborator or has had any other professional relationship with any person on the package who has a major role
-* The potential reviewer/editor serves as a member of the advisory board for the project under review
-*  The potential reviewer/editor would receive a direct or indirect financial benefit if the package is accepted
+* The potential reviewer/editor are from the institution or institutional component (e.g., department) of any author with a major role.
```
Comment (mpadge) [inline softwarereview_policies.Rmd:38]:
```suggestion
* The potential reviewer/editor are from the same institution or institutional component (e.g., department) as any author with a major role. 
```

File: softwarereview_policies.Rmd [hunk 2]
```diff
@@ -35,10 +35,10 @@ Last but not least, you'll find the [code of conduct of rOpenSci Software Peer R
 
 Following criteria are meant to be a guide for what constitutes a conflict of interest for an editor or reviewer. The potential editor or reviewer has a conflict of interest if:
 
-* The authors with a major role are from the potential reviewer/editor's institution or institutional component (e.g., department)
-* Within in the past three years, the potential reviewer/editor has been a collaborator or has had any other professional relationship with any person on the package who has a major role
-* The potential reviewer/editor serves as a member of the advisory board for the project under review
-*  The potential reviewer/editor would receive a direct or indirect financial benefit if the package is accepted
+* The potential reviewer/editor are from the institution or institutional component (e.g., department) of any author with a major role. 
+* The potential reviewer/editor has been a collaborator or has had any other professional relationship with any person on the package who has a major role within in the past three year.
```
Comment (mpadge) [inline softwarereview_policies.Rmd:39]:
```suggestion
* The potential reviewer/editor has been a collaborator or has had other professional relationships with at least one person on the package who has a major role within in the past three year.
```

File: softwarereview_policies.Rmd [hunk 3]
```diff
@@ -35,10 +35,10 @@ Last but not least, you'll find the [code of conduct of rOpenSci Software Peer R
...
+* The potential reviewer/editor serves as a member of the advisory board for the project under review.
+* The potential reviewer/editor would receive a direct or indirect financial benefit if the package is accepted.
```
Comment (mpadge) [inline softwarereview_policies.Rmd:41]:
```suggestion
* The potential reviewer/editor would receive a direct or indirect financial benefit if the package were accepted.
```

File: softwarereview_policies.Rmd [hunk 4]
```diff
@@ -35,10 +35,10 @@ Last but not least, you'll find the [code of conduct of rOpenSci Software Peer R
 
 Following criteria are meant to be a guide for what constitutes a conflict of interest for an editor or reviewer. The potential editor or reviewer has a conflict of interest if:
 
-* The authors with a major role are from the potential reviewer/editor's institution or institutional component (e.g., department)
-* Within in the past three years, the potential reviewer/editor has been a collaborator or has had any other professional relationship with any person on the package who has a major role
-* The potential reviewer/editor serves as a member of the advisory board for the project under review
-*  The potential reviewer/editor would receive a direct or indirect financial benefit if the package is accepted
+* The potential reviewer/editor are from the institution or institutional component (e.g., department) of any author with a major role. 
+* The potential reviewer/editor has been a collaborator or has had any other professional relationship with any person on the package who has a major role within in the past three year.
+* The potential reviewer/editor serves as a member of the advisory board for the project under review.
```
Comment (mpadge) [inline softwarereview_policies.Rmd:40]:
```suggestion
* The potential reviewer/editor serves, or has served, as a member of the advisory board for the project under review.
```

**Review by mpadge — CHANGES_REQUESTED** [review]
Thanks @eliocamp. Please ping again once you've accepted those minor suggestions (with modification if you like), and we'll happily approve.

**Comment by eliocamp** [comment]
@mpadge There you go, all accepted. :)

**Comment by mpadge** [comment]
One more request @eliocamp: could you please add a line desribing these changes to the [changelog](https://github.com/ropensci/dev_guide/blob/main/booknews.Rmd) as well? Let us know, and then I'll approve and merge. Thanks

**Comment by eliocamp** [comment]
Ready! Let me know if it looks good.

**Review by mpadge — APPROVED** [review]
Great, thanks!

**Pull request merged**